node/src/test/scala/coop/rchain/finalization/FinalizationSpec.scala

```diff
@@ -7,7 +7,7 @@ import cats.{Applicative, Foldable, Monad}
 import coop.rchain.casper.api.GraphzGenerator.{DagInfo, ValidatorsBlocks}
 import coop.rchain.casper.api.ValidatorBlock
 import coop.rchain.graphz._
-import coop.rchain.sdk.simulation.data.Finalization02._
+import coop.rchain.casper.pCasper.sim.Simulation._
```

Comment:
What is the meaning of _pCasper_?

Usually package names are lowercase. 

> Scala packages should follow the Java package naming conventions ...
https://docs.scala-lang.org/style/naming-conventions.html#packages

> Package names are written in all lower case to avoid conflict with the names of classes or interfaces.
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
Partial Casper, or Partitioned Casper